english/cowboy/cowboy.md

```diff
@@ -1,5 +1,8 @@
 ... Once you feel your best bib and tucker are all correct, you strut into the
 bar, push the nearest deadbeat and start the meanest brawl the ruff'n'tumble
 tavern has ever seen.
-You grab your aggressor and ask him "Where are we ?", he 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.